docs/11/index.md

```diff
@@ -807,57 +832,60 @@ You should see a visualization like this:
 The top new line is set of labels being used as inputs into
 the justification. In this case there's only one label;
 the label you see may be different than what's shown here.
-Under that is a copy of that statement:
+Under that is a copy of each statement from each label, which in this case
+is one statement:
 
 > `|- 2 = ( 1 + 1 )`
 
 Notice that variable parts of this statement are boxed
 and directed lines connect them to another statement below.
-The statement below is the set of required patterns
-required by `oveq2i`, in this case `|- A = B`.
-The lines show that in this use of `oveq2i`, `A` will be
-`2`, and `B` will be `( 1 + 1 )`.
 
-The result of `oveq2i` are `( C F A ) = ( C F B )`.
+In center of a visualization is always a sequence with zero or statements
```

Comment:
`with zero or statements` -> `with zero or few statements`

Reply:
Technically it's "zero or more". I wouldn't want to have a lot, but there's no *formal* reason it must be few.